src/Xamarin.Android.Build.Tasks/Xamarin.Android.Legacy.targets

```diff
@@ -299,6 +299,10 @@ projects. .NET 5 projects will not import this file.
           Include="$(OutDir)$(TargetFileName)"
           Condition="Exists ('$(OutDir)$(TargetFileName)')"
       />
+      <FilteredAssemblies
+        Include="$(_GenerateResourceDesignerAssemblyOutput)"
```

Comment:
This should be indented another two spaces, matching the `<FilteredAssemblies/>` 3 lines down.